src/components/UserInfo.js

```diff
@@ -17,9 +18,19 @@ export default class UserInfo {
     };
   }
 
+  avatarUserInfo() {
```

Comment:
Function names should start with a **verb** in infinitive (there is such a point in the checklist). It’s common in coding to do it so that functions could `speak` for themselves about what they do.


`getAvatarUrl`